pkg/capabilities/v2/actions/http/action.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/actions/http";
+
+import "values/pb/values.proto";
+import "capabilities/v2/protoc/pkg/pb/cre_metadata.proto";
+import "workflows/sdk/v2/pb/sdk.proto";
+
+message Inputs {
+  string url = 1;
+  string method = 2;
+  map<string, string> headers = 3;
+  bytes body = 4;
+  int32 timeout_ms = 5;
+}
+
+message Outputs {
+  string error_message = 1;
+  uint32 status_code = 2;
+  map<string, string> headers = 3;
+  bytes body = 4;
+}
+
+service Client {
+  option (cre.metadata.capability) = {
+    mode: Node
+    capability_id: "http-actions@1.0.0"
```

Comment:
let's start at v0.1.0. Until we're ready to release, I want to stay away from v1.0.0. If you don't now, I'll change it when we're going to the protos repo :).